src/simplewallet/simplewallet.cpp

```diff
@@ -969,12 +983,7 @@ bool simple_wallet::finalize_multisig(const std::vector<std::string> &args)
     return true;
   }
 
-  const auto orig_pwd_container = get_and_verify_password();
-  if(orig_pwd_container == boost::none)
-  {
-    fail_msg_writer() << tr("Your original password was incorrect.");
-    return true;
-  }
+  SCOPED_WALLET_UNLOCK();
```

Comment:
The password input to `finalize_multisig` is mandatory, while `SCOPED_WALLET_UNLOCK` skips the password input when `ask-password` is disabled. Fix:
```c++
  const auto orig_pwd_container = get_and_verify_password();
  if(orig_pwd_container == boost::none)
  {
    fail_msg_writer() << tr("Your original password was incorrect.");
    return true;
  }
  tools::wallet_keys_unlocker unlocker(*m_wallet, orig_pwd_container);
```